projects/feature-activation/0004-phased-testing.md

```diff
@@ -0,0 +1,206 @@
+- Feature Name: Feature Activation Phased Testing
+- Start Date: 2023-08-01
+- Initial Document: [Feature Activation](https://docs.google.com/document/d/1IiFTVW1wH6ztSP_MnObYIucinOYd-MsJThprmsCIdDE/edit)
+- Author: Gabriel Levcovitz <<gabriel@hathor.network>>
+
+# Summary
+[summary]: #summary
+
+This RFC describes a phased procedure for testing the Feature Activation project.
+
+# Motivation
+[motivation]: #motivation
+
+After all changes introduced by the previous Feature Activation RFCs are implemented, the mechanism should be tested with some "fake" features in both the testnet and mainnet environments, and only then we'll be able to use it with confidence in the mainnet to change real features like updating the maximum merkle path length, for example.
+
+# Guide-level explanation
+[Guide-level explanation]: #guide-level-explanation
+
+The testing will consist of 3 phases:
+
+1. NOP features in testnet
+2. Maximum merkle path length update in testnet
```

Comment:
We shouldn't change it in the testnet if we don't plan to change it soon in the mainnet. I mean, testnet is a testing network that must mirror mainnet behavior.

Reply:
I guess we do plan to change it soon on the mainnet, right? We could do it just after this testing process is complete and validated. If for any reason the testing fails on testnet and we need more time, we can always revert the change on testnet.

Reply:
In this case, deploying on testnet shouldn't be part of the rollout of the change on the maximum merkle path project? If it's simple enough, we can do everything on this project, so we will deploy this change on both the testnet and the mainnet.

Reply:
As discussed on meeting, the maximum merkle path length update for both testnet and mainnet will be described on a separate RFC. I left the original testnet text about it though, as reference.

Reply:
Thanks!